### ISSUE TITLE
Problems with unicode width

### ISSUE THREAD

**Issue by ilyagr** [slowli/term-transcript#51]
Let's say I have a file named `aa` with the following content:

```
@    OYRLTR ilyagr@ now 666180
├─╮  (empty) merged
◉ │  XTWULW ilyagr@ now second 020424
│ │  second
│ ◉  QUMUSS ilyagr@ now third 41c3f6 conflict
```

If you run `term-trascript exec "cat aa"`, you get something similar to what Github shows:

![aa](https://user-images.githubusercontent.com/4123047/229266878-91407d77-1967-437a-ba12-1e3a053fcca2.svg)

However, in a real terminal, the lines are perfectly aligned:
![image](https://user-images.githubusercontent.com/4123047/229266921-03af21e9-6651-4c32-8179-539cf0bf84da.png)

(This is alacritty, but it works the same on a few terminals).

---------------

I'm not sure if the SVG looks the same in all browsers on all devices. In my Chrome browser on a Chromebook, the beginning of this very message looks like this:

>![image](https://user-images.githubusercontent.com/4123047/229266988-536714d9-bac1-4303-afb3-346454f99854.png)

----------------

**Update:** Indeed, this is browser-dependent, and likely then depends on what fonts the browser happens to be using. Here's how the same snippet looks in Firefox. (Ignore the light theme; that's because I didn't log in to Github).

![image](https://user-images.githubusercontent.com/4123047/229310267-68990f65-b3ce-4838-b6dc-d3e1ce4e2e08.png)

**Comment by ilyagr** [comment]
https://github.com/asciinema/asciinema-player/issues/34 is a similar issue with asciinema. (There are a few related ones in that repo).

**Comment by ilyagr** [comment]
Thank you for the snippet! Given all the limitations, it makes sense not to try to address these problems with this tool.

A couple of notes that may or may not be helpful to people:

I know the Fira Code font contains all these symbols. I imagine Fira Mono (which is available on Google Fonts) does as well.

I did some research myself, and another relevant concept is that fonts can be [embedded in the svg file](https://vecta.io/blog/how-to-use-fonts-in-svg), but it's laborious to do manually and none of the tools I tried worked out of the box.
